tzrec/tools/create_fg_json.py

```diff
@@ -69,5 +85,22 @@
             reserves.append(column.strip())
         fg_json["reserves"] = reserves
 
-    with open(os.path.join(args.fg_output_dir, "fg.json"), "w") as f:
+    fg_path = os.path.join(args.fg_output_dir, "fg.json")
+    with open(fg_path, "w") as f:
         json.dump(fg_json, f, indent=4)
+
+    project = args.odps_project_name
+    fg_json_name = args.fg_resource_name
+    if project is not None and fg_json_name is not None:
+        account, odps_endpoint = _create_odps_account()
+        o = ODPS(
+            account=account,
+            project=project,
+            endpoint=odps_endpoint,
+        )
+        if o.exist_resource(fg_json_name):
+            o.delete_resource(fg_json_name)
```

Comment:
add an arg `force_update_resource` to control whether to overwrite the resource or not.